proto/sandbox.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+import "types.proto";
+
+option go_package = "proto";
+
+service Sandbox {
+    // Runs new Firecracker VM instance
+    rpc CreateVM(CreateVMRequest) returns (CreateVMResponse);
+
+    // Stops existing Firecracker instance by VM ID
+    rpc StopVM(StopVMRequest) returns (google.protobuf.Empty);
+
+    // Gets VM instance socket file location
+    rpc GetVMAddress(GetVMAddressRequest) returns (GetVMAddressResponse);
+}
+
+// CreateVMRequest specifies creation parameters for a new FC instance
+message CreateVMRequest {
+    // The number of dummy drives to reserve in advance before running FC instance.
```

Comment:
I'd vote for different ordering here, since we're defining now.

```protobuf
FirecrackerMachineConfiguration MachineCfg = 1;
string KernelImagePath = 2;
string KernelArgs = 3;
FirecrackerDrive RootDrive = 4;
repeated FirecrackerDrive AdditionalDrives 5;
repeated FirecrackerNetworkInterface NetworkInterfaces = 6;
int32 ContainerCount = 7;
```

That way we're grouping all of the explicit VM configuration first and adding the container-oriented components at the end.